tests/foreman/api/test_contentviewversion.py

```diff
@@ -33,960 +33,1013 @@
+    # There should be only 1 record returned
+    assert len(default_cv) == 1
+    # There should be only 1 version
+    assert len(default_cv[0].version) == 1
+    default_cv = default_cv[0].version[0]
+    return lce1, lce2, default_cv
+
+
+# Tests for content view version creation.
+
+
+@pytest.mark.tier2
+def test_positive_create(module_cv):
+    """Create a content view version.
+
+    :id: 627c84b3-e3f1-416c-a09b-5d2200d6429f
+
+    :expectedresults: Content View Version is created.
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Critical
+    """
+    # Fetch content view for latest information
+    cv = module_cv.read()
+    # No versions should be available yet
+    assert len(cv.version) == 0
+
+    # Publish existing content view
+    cv.publish()
+    # Fetch it again
+    cv = cv.read()
+    assert len(cv.version) > 0
+
+
+@pytest.mark.tier2
+def test_negative_create(module_org):
+    """Create content view version using the 'Default Content View'.
+
+    :id: 0afd49c6-f3a4-403e-9929-849f51ffa922
+
+    :expectedresults: Content View Version is not created
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Critical
+    """
+    # The default content view cannot be published
+    cv = entities.ContentView(organization=module_org.id, name=DEFAULT_CV).search()
+    # There should be only 1 record returned
+    assert len(cv) == 1
+    with pytest.raises(HTTPError):
+        cv[0].publish()
+
+
+# Tests for content view version promotion.
+
+
+@pytest.mark.tier2
+def test_positive_promote_valid_environment(module_lce_cv, module_org):
+    """Promote a content view version to 'next in sequence'
+    lifecycle environment.
+
+    :id: f205ca06-8ab5-4546-83bd-deac4363d487
+
+    :expectedresults: Promotion succeeds.
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Critical
+    """
+    # Create a new content view...
+    lce1, _, _ = module_lce_cv
+    cv = entities.ContentView(organization=module_org).create()
+    # ... and promote it.
+    cv.publish()
+    # Refresh the entity
+    cv = cv.read()
+    # Check that we have a new version
+    assert len(cv.version) == 1
+    version = cv.version[0].read()
+    # Assert that content view version is found in 1 lifecycle
+    # environments (i.e. 'Library')
+    assert len(version.environment) == 1
+    # Promote it to the next 'in sequence' lifecycle environment
+    promote(version, lce1.id)
+    # Assert that content view version is found in 2 lifecycle
+    # environments.
+    version = version.read()
+    assert len(version.environment) == 2
+
+
+@pytest.mark.tier2
+def test_positive_promote_out_of_sequence_environment(module_org, module_lce_cv):
+    """Promote a content view version to a lifecycle environment
+    that is 'out of sequence'.
+
+    :id: e88405de-843d-4279-9d81-cedaab7c23cf
+
+    :expectedresults: The promotion succeeds.
+
+    :CaseLevel: Integration
+    """
+    # Create a new content view...
+    _, lce2, _ = module_lce_cv
```

Comment:
Its used only one time in line 158, you directly use there as `module_lce_cv[1]`.